dts/arm/st/h7rs/stm32h7rs.dtsi

```diff
@@ -329,6 +329,10 @@
 				reg = <0x58023c00 0x400>;
 				clocks = <&rcc STM32_CLOCK(AHB4, 15U)>;
 			};
+
+			pinctrl_dummy: dummy_pins {
+				pinmux = <0>;
+			};
```

Comment:
Why adding these dummy nodes ?
Also, if modifying the original commit, please update the commit message and add you sign-off. Or better, don't touch the commit and add a new commit

Reply:
> the reason being when trying west build, i find it requires dma-names and a non-empty pinctrl in sail_a and sail_b to build now.

If this is the case, then we should fix sai and make a dedicated PR for that.

Reply:
@mariopaja Do you see any reason that would be required ?
@jw0785 Can you provide more details as you seem to be the only one impacted ?